website/redirects.js

```diff
@@ -5,4 +5,29 @@ module.exports = [
     destination: '/',
     permanent: true,
   },
+  {
+    source: '/docs/kubernetes/:path*',
+    destination: '/docs/platforms/kubernetes/:path*',
+    permanent: true,
+  },
+  {
+    source: '/docs/glossary',
+    destination: '/docs/resources/glossary',
+    permanent: true,
+  },
```

Comment:
🤔 I'm not following the redirects here:

I see https://www.waypointproject.io/docs/glossary as valid, but we want to redirect to `/docs/resources/glossary`?

Reply:
The changes are not yet live, but they are on main. See https://github.com/hashicorp/waypoint/pull/3161

Reply:
Oh <kbd>derp</kbd> I completely forgot about `stable-website` 

I see that here https://tip.waypointproject.io/docs/resources/glossary